.circleci/config.yml

```diff
@@ -32,7 +32,7 @@ jobs:
     resource_class: medium
     executor:
       name: go/default
-      tag: '1.19'
+      tag: "1.20"
```

Comment:
This obviously also needs a bump in the CLI....I'm currently checking with Hammerhead on this.

Reply:
Apparently the CLI's already being built with Go 1.20 👍